src/lib/bootloader/grub2pwd.rb

```diff
@@ -131,7 +131,7 @@ def disable
     end
 
     def encrypt(password)
-      result = Yast::Execute.locally("/usr/bin/grub2-mkpasswd-pbkdf2",
+      result = Yast::Execute.on_target("/usr/bin/grub2-mkpasswd-pbkdf2",
```

Comment:
well, another option I see is to have that mkpasswd installed locally as it does not need to be on target system if it is decided to split it from grub2. We need on target system mainly tools used to create config and install stage1 due to new kernel and new grub2 modules - so grub2-mkconfig and grub2-install.

In general I am not against change if it is properly tested during upgrade from old SLE...do we still support migration from SLE11?

Reply:
I decided to go this way because is simpler and I don't think grub2-mkpasswd-pbkdf2 will be split from the grub2 package in the foreseeable future.

That tool is included in the grub2 package even for SLE-11. So if you are upgrading such a system the tool is already in the target.